tests/test_revoke_authority_materialize_523.py

```
@@ -0,0 +1,512 @@
+def test_revoke_decree_bundled_authority_reuses_authority_changes(game):
+    """旨意捆带授权：主目标是承诺/旨意；授权副作用只走 authority_changes。"""
+    db, state, content = game
+    holder = _minister(db)
+    # 授权授予案卷（已颁 + 在持权）
+    grant_d = _eligible_dossier(db, state, holder, target_id="捆带事域")
+    authority_id = _grant(
+        db, state, content, holder, "专差督办", "issue:捆带事域", grant_d,
+    )
+    # 把授予案卷当作可撤成命目标（已颁旨意）
+    target_id = int(grant_d["id"])
+    # 确保可 breach：status 须 promulgated/executing
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)